environment.yml

```diff
@@ -1,12 +1,19 @@
 name: test-binder-sage
 
 channels:
-  - defaults
```

Comment:
The main thing of this PR is that `conda-forge` should always come before `defaults`.